gigareview/docker-compose.yml

```diff
@@ -2,10 +2,34 @@ version: '3.7'
 
 services:
 
+  web:
+    image: nginx:latest
+    ports:
+      - "9170:80"
+    volumes:
+      - ./:/app
+      - ./default.conf:/etc/nginx/conf.d/default.conf
+
+  application:
+    build:
+      context: .
+      dockerfile: ./common/Dockerfile
+    ports:
```

Comment:
On application servers, this block is unnecessary

```
    ports:
      - "9009:9000"
```

the reason is because
(1) all services inside a docker-compose have access to the same network and their internal exposed ports is accessible to each over 
(2) There no reason to access the application server from outside Docker (that's what `9009` allows), as only the web container service needs to acces it